extensions/tokenizers/rust/src/models/bert.rs

```diff
@@ -63,57 +65,8 @@ pub struct BertConfig {
     #[serde(default)]
     use_cache: bool,
     classifier_dropout: Option<f64>,
+    pub id2label: Option<HashMap<String, String>>,
     pub use_flash_attn: Option<bool>,
-    model_type: Option<String>,
-}
-
-impl Default for BertConfig {
-    fn default() -> Self {
```

Comment:
So we don't need all these default value any more?

Reply:
Added back.